File: muebleria/src/main/java/pe/com/muebleria/model/base/Cliente.java
package pe.com.muebleria.model.base;


import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Cliente extends BaseResultado
{	
	private String nombre;
	
	private String apellido;
	
	private Integer cod_cliente;
	
}

File: Mantenedores-main/app/src/main/java/com/example/mantenedores/entity/Cliente.kt
package com.example.mantenedores.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity(tableName = "tb_clientes")
data class Cliente(
    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "cod_cliente") val cod: Int = 0,
    @ColumnInfo(name = "nombre") val nombre: String?,
    @ColumnInfo(name = "apellido") val apellidos: String?): Serializable{

    override fun toString(): String{
        return "Cliente(id=$cod, nombre='$nombre', apellidos='$apellidos')"
    }
}


